r2dbc-mssql/src/test/java/com/google/cloud/sql/core/R2dbcMssqlIntegrationTests.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.sql.core;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.r2dbc.spi.ConnectionFactories;
+import io.r2dbc.spi.ConnectionFactory;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import reactor.core.publisher.Mono;
+
+@Ignore
+@RunWith(JUnit4.class)
+public class R2dbcMssqlIntegrationTests {
+
+  @Rule
+  public Timeout globalTimeout= new Timeout(20, TimeUnit.SECONDS);
+
+  private ConnectionFactory connectionFactory;
+
+  @Before
+  public void setUpConnection() {
+    this.connectionFactory =
+        ConnectionFactories.get(
```

Comment:
We've updated our other tests to use a connection pool for testing. Can you update this as well? Also, if you use environment variables to build the connection string, we can check to see if the tests work with the CI before merging. See [here](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/blob/master/r2dbc-mysql/src/test/java/com/google/cloud/sql/core/R2dbcMysqlIntegrationTests.java#L84) and [here](https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/pull/263/files#diff-aa701b353f647923b9fcf80435664101f904aa9dee087951795b5375fe2ceb59R48) for reference implementations